sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample03_Sessions.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests.Samples
+{
+    public class Sample03_Sessions : ServiceBusLiveTestBase
+    {
+        [Test]
+        public async Task SendAndReceiveSessionMessage()
+        {
+            await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: true))
+            {
+                string connectionString = TestEnvironment.ServiceBusConnectionString;
+                string queueName = scope.QueueName;
+                #region Snippet:ServiceBusSendAndReceiveSessionMessage
```

Comment:
Ohh, you are providing session example for README. I think you forgot to add this snippet in README.